alacritty/src/event.rs

```diff
@@ -1146,7 +1146,9 @@ impl input::Processor<EventProxy, ActionContext<'_, Notifier, EventProxy>> {
                     },
                     WindowEvent::DroppedFile(path) => {
                         let path: String = path.to_string_lossy().into();
-                        self.ctx.write_to_pty((path + " ").into_bytes());
+                        let mut pathstr: String = "'".to_owned();
```

Comment:
I'm not sure that we should quote them. But why we're not using brackated paste for dnd @chrisduerr ?

Reply:
Just simplicity I suppose. But dropping files with escape sequences in them into the terminal would certainly be an interesting attack vector.